manual/reconnection/README.md

```diff
@@ -1,11 +1,58 @@
 ## Reconnection
 
-*Coming soon... In the meantime, see the javadoc for [ReconnectionPolicy].*
+If the driver loses a connection to a node, it tries to re-establish it according to a configurable
+policy. This is used in two places:
 
-<!--
-TODO cover:
-- reconnection policy
-- scheduled reconnections vs. gossip events
--->
+* [connection pools](../pooling/): for each node, a session has a fixed-size pool of connections to
+  execute user requests. If one or more connections drop, a reconnection gets started for the pool;
+  each attempt tries to reopen the missing number of connections. This goes on until the pool is
+  back to its expected size;
```

Comment:
In driver 3 this works differently: there is a single reconnection at the node level, that starts when it's detected as DOWN, and will try to reopen all pools (one per active session).

If the pool is under capacity, this is handled on the fly in `borrowConnection`, but I don't think we need to get into as much detail here.